newsletter-october-2022.rst

```diff
@@ -0,0 +1,70 @@
+.. post:: Oct 12, 2022
+   :tags: newsletter, python
+   :author: Eric Holscher
+   :location: BND
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
+
+Read the Docs newsletter - October 2022
+=========================================
+
+September was exciting because a few members of our team finally got to gather in person.
+Manuel, Benjamin, and Eric all attended Djangocon Europe in September,
+and had lots of great discussions around documentation.
+
+Also, as we mentioned in Q4 we're going to be focusing on our core platform features.
+This means we'll have fewer new features to talk about, 
+but lots of smaller improvements to the overall experience of using Read the Docs.
+
+New features
+------------
+
+The latest updates from our team:
+
+- We fully rolled out support for auto-cancelling users when a user pushes to a branch. We wrote up a `full post <https://blog.readthedocs.com/cancel-old-builds/>`_ on the blog about it, where you can read more.
+- Redirects can now be edited directly in our dashboard. This was an oversight that we are glad to have finally fixed the UX around.
+- We fixed the bug we had previously mentioned around Gitlab tokens expiring. This caused PR builds for these users to stop properly reporting status. If you login with Gitlab again on our service, they will continue working indefinitely now.
```

Comment:
This isn't really part of the "New features" section. I checked other newsletter posts looking for "Bug fixes" section or similar but it seems that we don't have it. So, we should put it under the "Possible issues" maybe?

Reply:
I think that's probably true, but I think it's fine for now 🤷